notebooks/01_data_preprocessing.ipynb

```diff
@@ -22,7 +22,8 @@
     "import scipy as sp\n",
```

Comment:
<p>Save the Data? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/ERA5-dashboard-cookbook/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>